app/slashcommands/auto_constants.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 const (
-	UserPassword         = "passwd"
+	UserPassword         = "Usr@MMTest123"
```

Comment:
Why change the password?

Reply:
It was failing and returning an error saying password needs to contain some characters. I traced it to a password validation in the code